install/gcp/upgrade_tools/db_migrator.py

```diff
@@ -67,24 +56,35 @@ def alter_column(table, old_column, new_column):
         new_column (sqlalchemy.schema.Column): The sql alchemy column object,
             this is the column to update to.
     """
-    LOGGER.info('Attempting to alter column: %s', old_column.name)
+    LOGGER.info(f'Attempting to alter column {table.name}.{old_column.name}.')
 
     # bind the old column with the corresponding table.
     old_column.table = table
-
     old_column.alter(name=new_column.name,
                      type=new_column.type,
                      nullable=new_column.nullable)
 
 
+def create_column(table, column):
+    """Create Column.
+
+    Args:
+        table (sqlalchemy.schema.Table): The sql alchemy table object.
+        column (sqlalchemy.schema.Column): The sql alchemy column object.
+    """
+    if column.name not in table.columns:
```

Comment:
Main functional change. The rest is logging and re-ordering the methods alphabetically.